### ISSUE TITLE
Support for horizontal dragging

### ISSUE THREAD

**Issue by dev-bre** [justasm/DragLinearLayout#33]
Any idea how to update this layout to support horizontal dragging as well? It could be really useful in case of tablets where the screen goes more horizontally than vertically.

**Comment by justasm** [comment]
I agree. See #10, #12 and #20 for previous discussions regarding this issue.

Also check out the work of @cmcneil and @meesec in their respective forks for some inspiration.

https://github.com/cmcneil/DragLinearLayout
https://github.com/meesec/DragLinearLayout
